### PR TITLE
[WIP] Add failing test for `<input placeholder={{emptyString}}>`

### PR DESCRIPTION
**Pull request by cibernox**
Let's see if I can fix this thing

### CODE REVIEW

**Comment by chadhietala** [comment]
I believe this going to be fixed by #319.

**Comment by cibernox** [comment]
@chadhietala Yes, that sounds like a fix.

File: packages/glimmer-runtime/lib/dom/props.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export function normalizeProperty(element, slotName) {
 }
 
 export function normalizePropertyValue(value) {
-  if (value === '') {
+  if (value === '') { // WAT? Why? Because of the distinction prop/attrs? Do we need a whitelist of props/attrs?
```
Comment (rwjblue) [inline packages/glimmer-runtime/lib/dom/props.ts]: Hehe. Wat indeed. 😀

Comment (rwjblue) [inline packages/glimmer-runtime/lib/dom/props.ts]: Can you remove this comment so the test can be cherry-picked into #319?

Comment (cibernox) [inline packages/glimmer-runtime/lib/dom/props.ts]: The test is in a standalone commit already. Just pick 24794e5

Comment (rwjblue) [inline packages/glimmer-runtime/lib/dom/props.ts]: Gotcha, I didn't spot that, thank you!

**Pull request closed without merging**